ovh/data_cloud_project_region_loadbalancer_log_subscription_test.go

```diff
@@ -0,0 +1,51 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccCloudProjectLoadBalancerGetLogSubscription_basic(t *testing.T) {
+	config := fmt.Sprintf(testAccCloudProjectSubscription,
+		os.Getenv("OVH_CLOUD_PROJECT_SERVICE_TEST"),
+		os.Getenv("OVH_CLOUD_PROJECT_WORKFLOW_BACKUP_REGION_TEST"),
+		os.Getenv("OVH_CLOUD_LOADBALANCER_ID_TEST"),
+		os.Getenv("OVH_CLOUD_STREAM_ID_TEST"),
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheckCloud(t)
+			testAccCheckCloudProjectExists(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ovh_cloud_project_region_loadbalancer_log_subscription.test", "created_at"),
```

Comment:
you should check more fields, for example the `kind` that you set explicitly to `haproxy`